www/src/templates/template-starter-showcase.js

```diff
@@ -451,6 +451,12 @@ const styles = {
     color: colors.gatsby,
     textDecoration: `none`,
   },
+  screenshot: {
+    backgroundColor: "rgb(255, 255, 255)",
+    boxShadow: "rgb(102, 51, 153) 0px 0px 90px -24px",
```

Comment:
Hey Amberley! 👋
Thanks for taking this on!

Please use values from https://github.com/gatsbyjs/gatsby/blob/0c88a329d1d77cda2dbb8eac78af3be237d9e75d/www/src/utils/colors.js#L60 if possible.
The equivalent to `rgb(102, 51, 153)` here is `colors.gatsby` and can be used like this: https://github.com/gatsbyjs/gatsby/blob/0c88a329d1d77cda2dbb8eac78af3be237d9e75d/www/src/templates/template-starter-showcase.js#L468

Reply:
I also noticed that the `box-shadow` for the starter detail screenshot is a little bigger and more opaque than the one we're using for the showcase-site detail screenshots: https://github.com/gatsbyjs/gatsby/blob/0c88a329d1d77cda2dbb8eac78af3be237d9e75d/www/src/components/showcase-details.js#L495

I like yours, too, though 👍—I only fear that too much purple might conflict with some of the more colorful screenshots. Maybe we can find something in between the two?

Reply:
Whereever we end up regarding that `box-shadow` intensity/visibility 😄😉, I think this is a good opportunity to consolidate the styles for the two screenshots.

We already have a file for CSS used in both site- and starter-showcases: [`src/views/shared/styles.js`](https://github.com/gatsbyjs/gatsby/blob/0c88a329d1d77cda2dbb8eac78af3be237d9e75d/www/src/views/shared/styles.js)—could you please add a new entry to `styles` there, and use it in both `src/templates/template-starter-showcase.js` and also replace the site showcase detail screenshot styles `src/components/showcase-details.js`?